core/src/CopyChip/CopyChip.js

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useEffect } from "react";
+import cx from "classnames";
+
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { faClipboard, faCheck } from "@fortawesome/free-solid-svg-icons";
+
+import { Button } from "../Button";
+
+import styles from "./CopyChip.less";
+
+export const CopyChip = (props) => {
+  const defaultCopySuccessMessage = "Copied";
+  const { copySuccessMessage = defaultCopySuccessMessage, value } = props;
+
+  const [showCopySuccess, setCopySuccess] = useState(false);
+
+  function fallbackToCopy(text) {
+    if (window.clipboardData && window.clipboardData.setData) {
+      // IE specific code path to prevent textarea being shown while dialog is visible.
+      return window.clipboardData.setData("Text", text);
+    } else if (
+      document.queryCommandSupported &&
+      document.queryCommandSupported("copy")
+    ) {
+      const textarea = document.createElement("textarea");
+      textarea.innerText = text;
+      const parentElement = document.getElementById("copy");
```

Comment:
This component could easily be used more than once in a view. IDs should semantically only appear once in a document. Beyond semantics if this is present in a view more than once this selection could end up being in correct.